sass/blocks/developer/_developer.scss

```diff
@@ -1,26 +1,50 @@
 .developer {
-  &__paragraph {
+  &__svg {
+    position: absolute;
+    top: 13px;
+    left: 83px;
+  }
+
+  &__logo {
+    display: inline-block;
+    position: relative;
+
+    &:hover .developer__svg {
+      fill: $color-blue;
+    }
+
+    &:active .developer__svg {
+      opacity: 0.3;
+    }
+  }
+
+  &__text {
     display: inline-block;
+    position: relative;
+    top: 16px;
```

Comment:
I am not really fond of how you positioned it. Use rather explicit top paddings or margins and right padding instead of -20 left.